{{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/apps/settings/static/settings/js/admin/fields.js

```diff
@@ -9,7 +9,8 @@ function hide_inline_section_fields () {
     '.field-string': ['string'],
     '.field-text': ['text'],
     '.field-number': ['number'],
-    '.field-image': ['image']
+    '.field-image': ['image'],
+    '.field-html': ['html'],
```

Comment:
lint:js hasn't picked up on this, but we don't usually have dangling commas in JS